docs/en/developers/01_First_steps.md

```diff
@@ -1,6 +1,47 @@
-# Environment configuration
+# Environment configuration (Docker)
 
-**TODO**
+FreshRSS is built with PHP and use a homemade framework, Minz. The dependencies are directly included in the source code and so you don't need Composer.
+
+There are various ways to configure your development environment, the easiest and most supported one is based on Docker. This is the solution documented below. If you already have a working PHP environment, you probably don't need it.
+
+We assume here that you use a GNU/Linux distribution, capable to run Docker. Otherwise, you'll have to adapt the commands to your case.
+
+The commands that follow have to be executed in a console. They start by `$` when commands need to be executed as normal user, and by `#` when they need to be executed as root user. You don't have to type these characters. A path may be indicated before to help you to identify where they need to be executed. For instance, `app$ echo 'Hello World'` indicates that you have to execute `echo` command under `app/` directory.
+
+First, you need to install [Docker](https://docs.docker.com/install/linux/docker-ce/ubuntu/).
+
+Once you're done, clone the repository with:
+
+```console
+$ git clone git@github.com:FreshRSS/FreshRSS.git
+$ # or
+$ git clone https://github.com/FreshRSS/FreshRSS.git
+$ cd FreshRSS
+```
+
+Note that, if you want to contribute, you have to fork the repository first and clone your fork instead of the "root" one. Adapt the commands in consequence.
+
+Then, the only command you need to know is the following:
+
+```console
+$ make start
+$ # or if your user is not in the `docker` group
+$ SUDO=sudo make start
+```
+
+This might take some time while Docker download the image.
+
+**You can now access FreshRSS at [http://localhost:8080](http://localhost:8080).** Just follow the install process and select the SQLite database.
+
+You can stop the containers by typing `CTRL + C` or with the following command, in another terminal:
```

Comment:
```suggestion
You can stop the containers by typing <kbd>Control</kbd> + <kbd>c</kbd>` or with the following command, in another terminal:
```

Using `<kbd>` looks better (plus it provides semantics), lowercase c for consistency, see https://github.com/FreshRSS/FreshRSS/tree/125a83efc9e6cafb5eb274b07d1d006b0abb3031/Docker#development-mode

Reply:
I saw that in the other PR and told myself I should do the same here… but then I forgot about it!